api/src/models/AudioFile.ts

```diff
@@ -33,7 +48,7 @@ class AudioFile {
     }
 
     derivativePath(extension = "flac"): string {
-        const filename = path.basename(this.filename);
+        const filename = path.basename(this.filename, ".*");
```

Comment:
The Node basename function doesn't support ".*" -- it's a little more work, but you can copy my solution from here:

https://github.com/Geoffsc/VuDLPrepJS/blob/dev/api/src/models/ImageFile.ts#L61-L66